projects/cdk/directives/auto-focus/autofocus.options.ts

```diff
@@ -38,10 +42,17 @@ export const TUI_AUTOFOCUS_OPTIONS = new InjectionToken<TuiAutofocusOptions>(
 
 export function tuiAutoFocusOptionsProvider(
     options: Partial<TuiAutofocusOptions>,
-): ValueProvider {
+): FactoryProvider {
     return {
         provide: TUI_AUTOFOCUS_OPTIONS,
-        useValue: {...TUI_AUTOFOCUS_DEFAULT_OPTIONS, ...options},
+        useFactory: (isCypress: boolean) => {
```

Comment:
I don't think we should hook into `TUI_IS_CYPRESS` in a real code. Can we circumvent it inside our test environment?